release-common-lib/src/main/java/org/reactome/util/compare/DBObjectComparer.java

```diff
@@ -22,27 +26,37 @@
  */
 public class DBObjectComparer
 {
+	private static Map<GKInstance, Map<SchemaAttribute, List<Object>>> instanceAttributeToValuesMap = new HashMap<>();
+	private static Map<GKInstance, List<SchemaAttribute>> instanceToRegularAttributesMap = new HashMap<>();
+	private static Map<GKInstance, List<SchemaAttribute>> instanceToReferrerAttributesMap = new HashMap<>();
+
+	private static final int MAX_RECURSION_DEPTH = 5;
+	private static final int INITIAL_INSTANCES_DIFFERENCES_COUNT = 0;
+	private static final int INITIAL_RECURSION_DEPTH = 0;
+
```

Comment:
I would recommend prefixing these variable names with "DEFAULT_" since they can be overridden if users give their own values.

Reply:
Good call - done!

Reply:
On second thought, maybe "INITIAL" is not needed if "DEFAULT" is there? Not sure if that gets redundant or just too long.

Reply:
But why no "DEFAULT_" for MAX_RECURSION_DEPTH?

Reply:
Yeah, I think you're right.  I will remove "INITIAL" and replace it with "DEFAULT".  I'll also add it for "MAX_RECURSION_DEPTH" - when I made the change I only considered the ones that began with "INITIAL" and overlooked that one.

Reply:
Done!